src/plots/cartesian/graph_interact.js

```diff
@@ -778,8 +778,9 @@ function createHoverText(hoverData, opts) {
     var i, traceHoverinfo;
     for (i = 0; i < hoverData.length; i++) {
         traceHoverinfo = hoverData[i].trace.hoverinfo;
-        if (traceHoverinfo.indexOf('all')===-1 &&
-                traceHoverinfo.indexOf(hovermode)===-1) {
+        var parts = traceHoverinfo.split('+');
```

Comment:
Great. Let's keep that in mind when we get to https://github.com/plotly/plotly.js/issues/34. There are only a few `flaglist` attributes at the moment, but more are coming.